### PR TITLE
handbrake 1.3.2

### PR DESCRIPTION
**Pull request by miccal**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/handbrake.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Handbrake < Formula
   desc "Open-source video transcoder available for Linux, Mac, and Windows"
   homepage "https://handbrake.fr/"
-  url "https://download.handbrake.fr/releases/1.3.1/HandBrake-1.3.1-source.tar.bz2"
-  sha256 "c005de993af4123babbb60f7d12f6ff46fb7eea26ff0a3556034683901ce2c27"
+  url "https://github.com/HandBrake/HandBrake/releases/download/1.3.2/HandBrake-1.3.2-source.tar.bz2"
```
Comment (SMillerDev) [inline Formula/handbrake.rb:4]: Is this the official source now?

Comment (miccal) [reply Formula/handbrake.rb:4]: @SMillerDev I was not sure -- the link `https://download.handbrake.fr/releases/1.3.2/HandBrake-1.3.2-source.tar.bz2` is HTTP 404, and their website points to the GitHub link I used:

Screen Shot 2020-05-05 at 23 42 34

**Comment by miccal** [comment]
Happy to close in favour of https://github.com/Homebrew/homebrew-core/pull/54221

**Comment by miccal** [comment]
Same error as in https://github.com/Homebrew/homebrew-core/pull/54221:
```
==> brew audit --skip-style handbrake --online
==> FAILED
Error: 1 problem in 1 formula detected
handbrake:
  * Files were found with references to the Homebrew shims directory.
    The offending files are:
      bin/HandBrakeCLI
```

**Comment by SMillerDev** [comment]
Closing in favor of the other PR.

**Pull request closed without merging**